pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -859,9 +867,35 @@ func (c *ClusterConfig) SetClusterStatus(cluster *eks.Cluster) error {
 	c.Status.Endpoint = *cluster.Endpoint
 	c.Status.CertificateAuthorityData = data
 	c.Status.ARN = *cluster.Arn
+
+	for k, v := range cluster.Tags {
+		if k == DisableStackPrefixTag {
+			if disableStackPrefix, err := strconv.ParseBool(*v); err == nil {
```

Comment:
We have a thing called, `IsEnabled` or something like that in the `api` pkg.

Reply:
That only works for a bool pointer, in this case it's a string pointer that's getting parsed for a bool value. That said, I did only decide to use the string "true"/"false" so that it's easier for users to read it from the stack tags too. If we go with not having the disableStackPrefix tag at all on stacks where it's not true, then this could easily just be a check for the existence of that tags key.